libraries/redshop/src/Shipping/Tag.php

```diff
@@ -278,6 +284,33 @@ protected static function replaceShippingAddressStartEnd(&$templateHtml, $shippi
                 array($shippingAddress->phone, \JText::_('COM_REDSHOP_PHONE'))
             );
 
+            $shopId     = \RedshopEntityOrder::getInstance($shippingAddress->order_id);
+            $shopIdTrim = explode("|", $shopId->shop_id);
+			
+            if (!empty($OrderData->shop_id)) {
+                self::replaceTag(
+                    $shippingData,
+                    $shopIdTrim,
+                    array('{postnord_shop_name}'),
+                    array('<div style="border-bottom: 1px solid #d5d5d5;">
```

Comment:
could you please add a class here, and move the styling code to the css file @Rmh1978

Reply:
Done.

When added it to admin css, then i made some additions and changes to display the filter and search boxes in e.g. admin Order list.

I can see that gulp release do .min.css   versions, but it does not look like it create a redshop.admin.min.css
Should i do that manually? @NguyenBao10